.circleci/config.yml

```diff
@@ -788,6 +788,204 @@ jobs:
+                        - npm
+                        - << parameters.packageManager >>
+                steps:
+                    - run:
+                        background: true
+                        command: |
+                            cd code
+                            yarn local-registry --open
+                        name: Verdaccio
+                        shell: bash.exe
+                    - run:
+                        command: |
+                            cd code
+                            yarn wait-on tcp:127.0.0.1:6001
+                            yarn wait-on tcp:127.0.0.1:6002
+                        name: Wait on Verdaccio
+                        shell: bash.exe
+                    - run:
+                        command: |
+                            cd ..
+                            mkdir empty-<< parameters.template >>
+                            cd empty-<< parameters.template >>
+                            npm set registry http://localhost:6001
+                            npx storybook init --yes --package-manager npm
+                            npm run storybook -- --smoke-test
+                        environment:
+                            IN_STORYBOOK_SANDBOX: true
+                            STORYBOOK_DISABLE_TELEMETRY: true
+                            STORYBOOK_INIT_EMPTY_TYPE: << parameters.template >>
+                        name: Storybook init from empty directory (Windows NPM)
+                        shell: bash.exe
+            - when:
+                condition:
+                    equal:
+                        - yarn1
+                        - << parameters.packageManager >>
+                steps:
+                    - run:
+                        background: true
+                        command: |
+                            cd code
+                            yarn local-registry --open
+                        name: Verdaccio
+                        shell: bash.exe
+                    - run:
+                        command: |
+                            cd code
+                            yarn wait-on tcp:127.0.0.1:6001
+                            yarn wait-on tcp:127.0.0.1:6002
+                        name: Wait on Verdaccio
+                        shell: bash.exe
+                    - run:
+                        command: |
+                            cd ..
+                            mkdir empty-<< parameters.template >>
+                            cd empty-<< parameters.template >>
+                            npx storybook init --yes --package-manager yarn1
+                            yarn storybook --smoke-test
+                        environment:
+                            IN_STORYBOOK_SANDBOX: true
+                            STORYBOOK_DISABLE_TELEMETRY: true
+                            STORYBOOK_INIT_EMPTY_TYPE: << parameters.template >>
+                        name: Storybook init from empty directory (Windows Yarn 1)
```

Comment:
Officially we already don't support Yarn 1 actively. We should drop the testing of Yarn 1 from CI.